INSTALL.md

```diff
@@ -0,0 +1,84 @@
+# Installing device plugins to cluster
+
+## Install device plugins via a DaemonSet
+
+Each plugin can be installed via a DaemonSet. The install changes slightly based on the desired plugin. See install instructions per [plugin](README.md#plugins).
+
+Installing plugins via DaemonSets deployes them to the ```default``` (or currently active) namespace. Use kubectl's ```--namespace``` argument to change the deployment namespace.
+
+## Install device plugins via device plugin operator
+
+A more advanced install method is via device plugin operator. Operator configures plugin deployments based on the supplied device plugin CRDs (Custom Resource Definitions). See installation instructions in the [operator README](cmd/operator/README.md#installation).
+
+Operator installs device plugins to the ```inteldeviceplugins-system``` namespace.
+
+## Install with HELM charts
+
+Device plugins can also be installed to a cluster using the device plugin [operator Helm chart](https://github.com/intel/helm-charts/tree/main/charts/device-plugin-operator) (depending on cert-manager and NFD). Individual plugin projects are under https://github.com/intel/helm-charts/tree/main/charts/.
+
+As with the method above, these steps will install device plugin operator and plugins under ```inteldeviceplugins-system``` namespace.
```

Comment:
is it always the same namespace with helm or can you override with `helm install`?

Reply:
No, it's not hardcoded. If user chooses another namespace via **--namespace** then operator and plugins are installed under that namespace.